src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -143,31 +144,26 @@ private bool ValidateOptions(FileWorkItemsOptions options, SarifWorkItemContext
             return valid;
         }
 
-        private static bool ValidateHostUri(string hostUriString, SarifWorkItemContext workItemFilingConfiguration)
+        // Validates that the URI coming from either the commandline or configuration is valid for filing work items.  A null or empty 
+        // value is valid if and only if a plug-in will later provide the value.  This cannot be determined until plugins are loaded later,
```

Comment:
>plug-in will later provide the value [](start = 43, length = 36)

Have you verified that using the org/project from the ownership service works? i.e. is it OK to retrieve the values that late in the process. Because if it doesn't work, we could just require the values in the command line or config.
Alternatively, we could create an IOwnershipTransformer interface and add the existence of ownership transformer to the check.